biblio/references_all.bib

```diff
@@ -11044,3 +11044,19 @@ @article{Vandel_Cassan_Lebre_Lecellier_Brehelin19
 number = {1},
 page = {103}
 }
+
+
+@inproceedings{Rendle10,
+  author    = {Steffen Rendle},
+  title     = {Factorization Machines},
+  booktitle = {{ICDM} 2010, The 10th {IEEE} International Conference on Data Mining,
+               Sydney, Australia, 14-17 December 2010},
+  pages     = {995--1000},
+  year      = {2010},
+  crossref  = {DBLP:conf/icdm/2010},
```

Comment:
PLease, can you remove the useless lines: crossref, url, timestamp, biburl, bibsource.